packages/ember-views/tests/views/text_field_test.js

```diff
@@ -553,7 +553,6 @@ test('should not reset cursor position when text field receives keyUp event', fu
 
   appendView(view);
 
-  view.$().val('Brosiedoon, King of the Brocean');
```

Comment:
I much prefer the new logic of caching the last value. IMHO, it has been fairly clear thst third parties changing the DOM underneath is not acceptable. It may have worked in this particular instance, but many many others did not.